### PR TITLE
💄  Issue #23: Inline code elements in header elements look small

### PR DESCRIPTION
**Pull request by macdonst**
Fixes Issue #23 

Not in love with the CSS change but it changes:

Screen Shot 2021-10-04 at 10 19 29

to look like:

Screen Shot 2021-10-04 at 10 20 51

### CODE REVIEW

**Review by tbeseda — APPROVED** [review]
LGTM
The only possible improvement might be using a variable for these specific sizes, but that would change how styles.css is created. Maybe something for upstream CSS later

**Comment by macdonst** [comment]
@tbeseda yeah, I couldn't figure out how to do this when a `code` tag is inside a header tag. I'm hoping @kristoferjoseph will have a better idea.

**Comment by kristoferjoseph** [comment]
What about?

```
code {
 font-size: inherit;
}
```

File: public/css/index.css [hunk 1]
```diff
@@ -81,6 +81,18 @@ code {
...
+h2 > code {
+  font-size:1.777rem;
+}
+h3 > code {
+  font-size: 1.333rem;
+}
+h4 > code, h5 > code {
+  font-size: 1rem;
+}
 .menu-transition {
```
Comment (kristoferjoseph) [inline public/css/index.css:96]: `
code {
  font-size: inherit;
}
`

But only if you are trying to match the size of the heading.

Comment (macdonst) [reply public/css/index.css:73]: Oh yeah, that's way better.

**Pull request merged**